test/binary_search_tree_test.rb

```diff
@@ -88,4 +88,24 @@ def test_it_can_insert_a_twelfth_node
     assert_equal 3, result
   end
 
+  def test_include?
+    @tree.insert(61, "Bill & Ted's Excellent Adventure")
+
+    assert @tree.include?(61)
+  end
+
+  def test_depth_of_single_node
+    @tree.insert(61, "Bill & Ted's Excellent Adventure")
+
+    assert_equal 1, @tree.depth_of(61)
+  end
+
+  def test_depth_of_several_nodes
+    @tree.insert(61, "Bill & Ted's Excellent Adventure")
```

Comment:
all of the copying `@tree.insert(int, "title")` seems ripe for a `setup` method.

Reply:
I created the setup method by using a hash with the score as the keys and the movie title and  its depth (both in an array), as the values. Then I use .each to iterate through the hash and do the insertions or assertions.